src/service/viewport-impl.js

```diff
@@ -1093,22 +1137,37 @@ export class ViewportBindingNaturalIosEmbed_ {
   }
 
   /** @override */
-  updatePaddingTop(paddingTop, opt_updateScrollPos, opt_lastPaddingTop) {
+  hideViewerHeader(transient, lastPaddingTop) {
+    if (!transient) {
+      this.updatePaddingTop(0);
+    } else {
+      // Add extra paddingTop to make the content stay at the same position
+      // when the hiding header operation is transient
+      onDocumentReady(this.win.document, doc => {
+        const existingPaddingTop =
+            this.win./*OK*/getComputedStyle(doc.body)['padding-top'] || '0';
```

Comment:
`existingPaddingTop` is the the padding top defined by the user css. `lastPaddingTop` and `paddingTop` are both extra paddingTop that doesn't include the default user-defined padding-top on body. I could rename `existingPaddingTop` to `userDefinedPaddingTop`